.goreleaser.yaml

```diff
@@ -0,0 +1,30 @@
+# This is an example .goreleaser.yml file with some sensible defaults.
+# Make sure to check the documentation at https://goreleaser.com
+before:
+  hooks:
+    - go mod tidy
+builds:
+  - main: main.go
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - windows
```

Comment:
We don't really support windows. For now I would take this out. The container fails starting and building on windows anyways. So that would probably fail the release.